src/main/webapp/script.js

```diff
@@ -312,7 +325,9 @@ function getGraduationRate(data) {
 
 /** Adds comments to page. */
 function loadComments() {
```

Comment:
Perhaps another naming nit on this - you're now trying to load comments and prepare the comment form for new submissions; I think it's a good idea to change the name of this to something like `loadAndPrepareComments`. Or similar. :) Another option (maybe a better one?) is to again avoid onload on the body, include this script at the bottom of the page instead, and then you can just call the methods individually in some sort of `init()` or similar function call.

function init() {
  const schoolId = getSchoolIdFromUrl();
  loadComments(schoolId);
  prepareCommentsForm(schoolId);
  ...
}

init();